launch-or-update-azure.sh

```diff
@@ -1,18 +1,37 @@
 #!/bin/bash
 
-if [ $# -neq 1  ]
-  echo "usage ./launch-or-update-azure.sh <namespace>"
+if [ $# -ne 4  ]; then
+  echo "usage ./launch-or-update-azure.sh <namespace> <azure sql server> <azure sql user> <azure sql pass>"  
   exit
 fi
 
 namespace=$1
+dbServer=$2
+saUserName=$3
+saPassword=$4
 basePath="$namespace-dev.buyingcatalogue.digital.nhs.uk"
 baseUrl="https://$basePath"
 baseIdentityUrl="$baseUrl/identity"
+dbName=bc-$namespace
+
+#helm dependency update src/buyingcatalogue/
+
+sed "s/REPLACENAMESPACE/$namespace/g" environments/azure-namespace-template.yml > namespace.yaml
+cat namespace.yaml
+kubectl apply -f namespace.yaml
 
 #helm upgrade bc gpitfuturesdevacr/buyingcatalogue -n $namespace -i -f environments/azure.yaml --debug \
-helm upgrade bc src/buyingcatalogue -n $namespace -i -f environments/azure.yaml \
+helm upgrade bc src/buyingcatalogue -n $namespace -i -f environments/azure.yaml --debug \
```

Comment:
Should the default be running in debug mode?

Reply:
removed debug